client/package.json

```diff
@@ -7,7 +7,8 @@
     "@babel/plugin-transform-react-jsx-self": "^7.10.4",
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
-    "@mui/material": "^5.11.11",
+    "@mui/lab": "^5.0.0-alpha.122",
```

Comment:
I think the `@mui/lab` package is only necessary for the tabs implementation

I'm okay adding it for future use if that is the intention

Reply:
Nice catch! I will work on it and request review again.